tuf/tuf.go

```diff
@@ -173,6 +173,15 @@ func (tr *Repo) RemoveBaseKeys(role string, keyIDs ...string) error {
 	return nil
 }
 
+// GetAllRoles returns a list of all role entries for this TUF repo
+func (tr *Repo) GetAllRoles() ([]*data.Role, error) {
+	roles := tr.keysDB.GetAllRoles()
+	if len(roles) == 0 {
+		return nil, data.ErrNoRoles{}
```

Comment:
I agree that `GetAllLoadedRoles` clarifies the distinction between what's actually loaded and what must be a valid TUF repo -- unless there are any objections I'll update the function name and return an empty array here instead of `data.ErrNoRoles{}`